src/videostreamindex/DefaultVideoStreamIndex.ts

```diff
@@ -67,12 +74,23 @@ export default class DefaultVideoStreamIndex implements VideoStreamIndex {
     this.currentIndex = indexFrame;
     this.streamToAttendeeMap = null;
     this.streamToExternalUserIdMap = null;
-    this.ssrcToStreamMap = null;
+  }
+
+  integrateSubscribeFrame(): void {
```

Comment:
I went back and forth on the naming of this.   I landed on trying to keep it consistent with the concept of "integrate..." since it is called as a Protocol Message is sent or received.  No problem renaming it though.